hivemq-edge/src/test/java/com/hivemq/combining/DataCombinerManagerTest.java

```diff
@@ -1,112 +1,119 @@
-/*
```

Comment:
Why is the copyright removed?

Reply:
Argh, that's a new test, I replaced the old one because the naming was misleading and forgot to add the header.

Reply:
Fixed